PR.md

```diff
@@ -0,0 +1,50 @@
+# Summary
+- **Status**: `draft` <!-- `review`, `suspended` -->
+- **RFCs**: [Link to RFC](#./link/to/rfc), [Link to RFC](#./link/to/rfc), or $\emptyset$.
+- **Categories**: any of `protocol-units`, `networks`, `scripts`, `util`, `cicd`, or `misc`.
+
+<!--
+Add your summary text here. 
+ -->
+
+# Core Software Components
+
+<!--
+Describe core software components added or modified by the PR. 
+-->
+
+# Ancillary Software Components
+<!--
+Describe components added or modified while working on core software components.
+-->
+
+# Testing
+
+## e2e and Integration Tests
+
+### Test Plan
+<!---
+Describe how your PR is tested. 
+-->
```

Comment:
I'd opt for just a ### Test Plan section. 
If any new unit or e2e tests were added, they can be described in the same section. 

Reasoning: 

I'm not a fan of ### Test Plan and ### Test components. In many cases tests will not need to be added but simply passing CI will suffice for a small change. So in those cases, these sections end up being ignored / redundant. 

We want the PR.md to provide meaningful and useful guidelines, without being overly restrictive / brittle. 

We also want the sections to be filled out for every merge, even if with just small amounts of detail, so they should be flexible enough

Reply:
What about just?:

# Testing
...

Reply:
👍